src/backends/pyplot.jl

```diff
@@ -1010,15 +1036,11 @@ function _before_layout_calcs(plt::Plot{PyPlotBackend})
             PyPlot.PyCall.hasproperty(ax, axissym) || continue
             axis = sp[axissym]
             pyaxis = getproperty(ax, axissym)
-            if axis[:mirror] && letter != :z
```

Comment:
Mirror will not work for 3d as matplotlib does not support it. Mirror right now does not work properly at all, so it is best to fix this later in a separate PR